<launch>

<node name="osmosis_control_node" type="osmosis_control_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="teleop_node" type="teleop_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e" />
<node name="safety_pilot_node" type="safety_pilot_node" pkg="osmosis_control"/>
<node name="localization_node" type="localization_node" pkg="osmosis_control" />
<node name="graph_planner_node" type="graph_planner_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="joy_teleop_node" type="joy_teleop_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="joy_node" type="joy_node" pkg="joy" output="screen" launch-prefix="xterm -e"/>
<node name="mission_manager_node" type="mission_manager_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="emergency_shutdown_node" type="emergency_shutdown_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>

<node name="FTM_Manager_node" type="FTM_Manager_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="checkProhibitedArea_node" type="checkProhibitedArea.py" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="DM1_ProhibitedArea_node" type="DM1_ProhibitedArea_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="DM2_CmdNotUpdated_node" type="DM2_CmdNotUpdated_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="DM3_WrongCommand_node" type="DM3_WrongCommand_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="DM5_NodeCrash_node" type="DM5_NodeCrash_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>

<node name="R3_EmergencyStop_node" type="R3_EmergencyStop_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>

<!--node name="navigation_node" type="navigation_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/-->

</launch>
